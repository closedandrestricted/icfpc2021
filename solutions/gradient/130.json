{"vertices":[[16,45],[16,52],[18,67],[13,72],[20,36],[22,43],[23,55],[23,62],[20,75],[27,28],[27,35],[29,44],[29,51],[29,65],[27,72],[34,29],[34,34],[35,41],[35,55],[35,62],[39,34],[37,41],[35,48],[41,51],[37,56],[41,62],[45,29],[45,41],[43,48],[37,58],[44,58],[47,68],[52,28],[48,34],[44,55],[43,62],[50,55],[51,62],[58,25],[45,41],[43,48],[48,57],[55,59],[53,30],[51,37],[50,49],[55,54],[62,60],[69,62],[54,44],[47,43],[62,57],[69,59],[76,65],[53,46],[57,52],[73,54],[79,58]]}