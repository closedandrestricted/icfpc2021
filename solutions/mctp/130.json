{"vertices":[[61,53],[64,59],[55,51],[48,49],[58,58],[65,58],[69,54],[62,52],[45,42],[52,48],[51,55],[66,51],[62,57],[55,52],[51,46],[46,46],[57,51],[62,56],[56,53],[50,57],[43,52],[50,50],[55,57],[49,53],[43,57],[39,51],[35,51],[43,53],[50,51],[41,51],[48,51],[32,55],[29,49],[36,51],[43,50],[40,44],[41,50],[34,48],[22,48],[29,52],[36,50],[33,41],[35,48],[15,49],[22,49],[29,49],[26,43],[28,47],[21,45],[15,46],[22,48],[19,45],[23,51],[14,42],[15,49],[12,43],[17,53],[11,49]]}