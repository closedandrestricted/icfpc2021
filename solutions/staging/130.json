{"vertices":[[62,24],[27,46],[17,65],[17,72],[23,34],[23,41],[23,55],[23,62],[20,41],[44,59],[47,58],[29,44],[29,51],[29,65],[64,19],[55,34],[35,34],[35,41],[35,55],[35,62],[28,46],[41,30],[41,44],[41,51],[41,65],[21,79],[74,21],[48,34],[48,41],[48,55],[48,62],[13,77],[55,20],[54,30],[30,54],[54,51],[54,65],[28,76],[66,11],[60,34],[16,57],[60,55],[60,62],[66,23],[64,28],[63,57],[66,51],[66,65],[32,50],[56,28],[37,58],[72,55],[14,31],[84,58],[16,86],[22,82],[55,53],[41,54]]}