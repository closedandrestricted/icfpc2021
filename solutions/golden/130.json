{"vertices":[[18,45],[16,52],[18,67],[12,71],[23,34],[23,41],[23,55],[23,62],[19,74],[28,29],[30,36],[29,44],[29,51],[29,65],[26,71],[35,28],[37,34],[35,41],[35,55],[35,62],[38,34],[37,41],[35,48],[41,51],[37,56],[44,58],[45,30],[45,41],[43,48],[37,58],[44,58],[46,66],[52,29],[49,35],[44,55],[43,62],[50,55],[52,62],[57,24],[48,42],[44,48],[48,57],[55,59],[57,31],[53,37],[37,48],[43,52],[60,63],[67,65],[48,42],[41,43],[50,55],[57,57],[73,61],[46,48],[53,48],[64,58],[70,54]]}